### PR TITLE
WRN-5708, WRN-8317: Added support for multiple ilib resources paths and detecting changed resources

### PR DESCRIPTION
**Pull request by MikyungKim**
Enact-DCO-1.0-Signed-off-by: Mikyung Kim (mikyung27.kim@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
WRN-5708:
For webOS that supports multiple resources, we need to invalidate the cache of strings. We've added a timestamp in ilibmanifest.json file so that we could validate the resources.
WRN-8317:
Added support for loading given multiple resources paths, e.g. resources, resources_0,...

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
WRN-5708:
Checking timestamp of `ilibmanifest.json` file and remove the cache of the strings.
WRN-8317:
Retrieve the result of additional resources path and combine the result with the default resources path

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
WRN-5708 is dependent with WRN-8317 so the PR is combined.

### Links
[//]: # (Related issues, references)
WRN-5708, WRN-8317

### CODE REVIEW

File: packages/i18n/src/Loader.js [hunk 1]
```diff
@@ -221,10 +242,23 @@ EnyoLoader.prototype.loadFiles = function (paths, sync, params, callback, rootPa
...
+						// Overwrite the _root/path result
+						Object.assign(ret[ret.length - 1], json);
+					}
+				};
 
 				if (this.isAvailable(_root, path)) {
 					getSync(this._pathjoin(_root, path), handler);
+
+					if (this.addPaths && Array.isArray(this.addPaths)) {
+						for (let addedRoot of this.addPaths) {
```
Comment (gouniLee) [inline packages/i18n/src/Loader.js:256]: fyi. addPath() takes "String" or "Array". so If ILIB_ADDITIONAL_RESOURCES_PATH value is a string, this.addPaths array length 1 always here.

File: packages/i18n/src/Loader.js [hunk 2]
```diff
@@ -221,10 +242,23 @@ EnyoLoader.prototype.loadFiles = function (paths, sync, params, callback, rootPa
 					}
 				};
 
-				this.loadManifestsSync(_root);
+				const handleAdditionalResourcesPath = (json, err) => {
+					if (found && !err && typeof json === 'object') {
```
Comment (gouniLee) [inline packages/i18n/src/Loader.js:246]: Question) doesn't need to consider the returning `text` type case here?
somethig like: `if (typeof(text) === "string") { text = JSON.parse(text);}`

File: packages/i18n/src/Loader.js (hunk 1 above)
Comment (MikyungKim) [reply packages/i18n/src/Loader.js:256]: I know. I wrote this for general usage.

File: packages/i18n/src/Loader.js (hunk 2 above)
Comment (MikyungKim) [reply packages/i18n/src/Loader.js:246]: We've already parsed the result from `getImpl` function.

File: packages/i18n/src/Loader.js [hunk 3]
```diff
@@ -50,6 +50,8 @@ const iLibResources = typeof ILIB_RESOURCES_PATH === 'undefined' ? '/locale' : I
 const cachePrefix = 'ENACT-ILIB-';
 const cacheKey = cachePrefix + 'CACHE-ID';
 const cacheID = typeof ILIB_CACHE_ID === 'undefined' ? '$ILIB' : ILIB_CACHE_ID;
+const timeStampKey = 'l10n_timestamp';
```
Comment (jeonghee27) [inline packages/i18n/src/Loader.js:53]: 10n?

Comment (MikyungKim) [reply packages/i18n/src/Loader.js:53]: It means localization.

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Pull request merged**